### PR TITLE
Fix README.md

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Fix README.md

### CODE REVIEW

**Comment by Andersson007** [comment]
ready for review

File: README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ If you encounter abusive behavior violating the [Ansible Code of Conduct](https:
 * Join the Ansible forum:
     * [ClickHouse Team](https://forum.ansible.com/g/ClickHouseTeam): by joining the team you will automatically get subscribed to the posts tagged with [clickhouse](https://forum.ansible.com/tag/clickhouse).
     * [Get Help](https://forum.ansible.com/c/help/6/none): get help or help others.
-    * [Posts tagged with 'postgresql'](https://forum.ansible.com/tag/clickhouse): leverage tags to narrow the scope.
+    * [Posts tagged with 'clickhouse'](https://forum.ansible.com/tag/clickhouse): leverage tags to narrow the scope.
```
Comment (oraNod) [inline README.md:100]:
```suggestion
    * [Posts tagged with 'clickhouse'](https://forum.ansible.com/tag/clickhouse): use tags to narrow the scope.
```

Comment (oraNod) [reply README.md:100]: IMO "leverage" is a bit too much when you can simply say "use".

Comment (Andersson007) [reply README.md]: simplicity is good, thanks, ready for review

**Review by oraNod — APPROVED** [review]

**Comment by oraNod** [comment]
Thanks @Andersson007

**Comment by Andersson007** [comment]
@oraNod thanks for reviewing and merging!

**Pull request merged**